ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/ExceptionRequestModal/ExceptionRequestForm.tsx

```diff
@@ -21,7 +21,11 @@ function getDefaultValues(cves: string[], scopeContext: ScopeContext): Exception
     const imageScope =
         scopeContext === 'GLOBAL'
             ? { registry: ALL, remote: ALL, tag: ALL }
-            : { registry: ALL, remote: scopeContext.image.name, tag: ALL };
+            : {
+                  registry: scopeContext.imageName.registry,
+                  remote: scopeContext.imageName.remote,
+                  tag: ALL,
+              };
```

Comment:
CVE deferral at the image level is for a single image name _and_ registry, not just a single image name across registries. This behavior was verified in VM 1.0